client/flags/flags.go

```diff
@@ -15,7 +15,7 @@ const (
 	// failures due to state changes that might occur between the tx simulation
 	// and the actual run.
 	DefaultGasAdjustment = 1.0
-	DefaultGasLimit      = 200000
+	DefaultGasLimit      = 300000
```

Comment:
PowerReduction will not be changed often, Is there any other way to store it on global variable and update it on every EndBlocker? cc. @sunnya97 @ValarDragon @psaradev

Reply:
Wdym?

Reply:
I'm confused as to why the gas limit is increased here

Reply:
gas limit is increased to bypass the issue on staking module cli transactions as staking params size has grown.

Reply:
Hmm...  @psaradev do you know which command fails with the old gas limit?

Reply:
MsgRedelegation looks fails, but it's not clear which line of code use power reduction directly or indirectly.

Reply:
I used to have this problem for changepubkey stuff when adding a param to auth params.

Reply:
@antstalepresh how did you solve it?

Reply:
If its only msg redelegation, can we override gas for only that tx? (It potentially was just at the limit before, and the param query put it over)

Reply:
Increased gas limit for only redelegation. https://github.com/sikkatech/cosmos-sdk/pull/65/commits/2ce6d4bcba5aa59e17d356fab3a99fbee2d240ee